content/en/docs/installation/guide/redhat-marketplace.md

```diff
@@ -0,0 +1,309 @@
+  - /spinnaker/openshift-container-platform/red-hat-marketplace
+---
+
+{{% alert title="Note" %}}This document is intended for users who have purchased the Armory Red Hat Marketplace offering. It will not work if you have not purchased the Armory Operator.
+
+Please contact [Armory](mailto:hello@armory.io) if you're interested in a Red Hat Marketplace Private Offer.{{% /alert %}}
+
+## Overview
+
+The Armory Operator is a Kubernetes Operator for Spinnaker<sup>TM</sup> that makes it easier to install, deploy, and upgrade Spinnaker. 
+
+## What is the Red Hat Marketplace?
+
+The Red Hat Marketplace is available directly from your OpenShift web console and provides an open cloud catalog that makes it easier to discover and access certified software for container-based environments in public clouds. With automated deployment, software is immediately available to deploy on any Red Hat OpenShift cluster, providing a fast, integrated experience. Discover and buy certified software, and quickly deploy. Access open source and proprietary software, with responsive support, streamlined billing and contracting, simplified governance, and single-dashboard visibility across clouds. Built in partnership by Red Hat and IBM, this marketplace helps organizations deliver enterprise software and improve workload portability.
+
+## Before you begin
+
+{{% alert title="Note" %}}You must have the cluster administrator role to install the operators from the Red Hat Marketplace.{{% /alert %}}
+ 
+1. These instructions assume you already have a Kubernetes cluster available in OpenShift Container Platform v4.4+
+2. Browse to the [Red Hat Marketplace](https://marketplace.redhat.com/en-us?_ga=2.94000664.441560370.1604955054-123808014.1597373423) and log in or create a new account.
+3. Follow the [instructions](https://marketplace.redhat.com/en-us/documentation/clusters?&_ga=2.94000664.441560370.1604955054-123808014.1597373423#register-openshift-cluster-with-red-hat-marketplace) to register your OpenShift cluster with the Red Hat Marketplace.
+4. When prompted `Would you like to go back to the Red Hat Marketplace now? [Y/n]`, type `Y` and the Red Hat Marketplace page is opened in your browser.
+5. Click <b>My software</b> >  <b>Visit the Marketplace.</b>
+6. In the search bar, type <b>armory</b> to load the armory tile.
+7. Click Purchase or Free trial to get started. From the Purchase complete page, click Install now. This installs the Armory operator into your cluster. Note that during the installation process you are required to select which OpenShift project to deploy the operator to from the Namespace scope drop-down list. After the operator is installed, your cluster connects back to Red Hat Marketplace and then becomes a target cluster for installing and managing the operator from Red Hat Marketplace.
+
+## Installation
+
+This document covers the following high-level steps:
+
+1. Configure application and pipeline configuration storage 
```

Comment:
```suggestion
1. Configuring application and pipeline storage 
```

So that you maintain the tenses in 2. and 3.

Also random note: in markdown you can create numbered lists with `1.` by itself, that way you don't have to renumber elements if you add/remove items from the list.

Example:
```markdown
1. this is
1. a numbered
1. list!
```

Rendered:

1. this is
1. a numbered
1. list!

Reply:
Thanks for the note @dogonthehorizon !